eel/eel-background.c

```diff
@@ -665,8 +665,15 @@ widget_realized_setup (GtkWidget     *widget,
     self->details->screen_monitors_handler =
         g_signal_connect (screen, "monitors-changed", G_CALLBACK (screen_size_changed), self);
 
-    self->details->use_common_surface =
-        (gdk_window_get_visual (window) == gtk_widget_get_visual (widget)) ? TRUE : FALSE;
+    if  (GDK_IS_X11_DISPLAY (display))
```

Comment:
One space after `if` should be sufficient.

Reply:
Found that, fixed it. Will push after the rest of these